statsmodels/tsa2/sarimax/estimators/tests/test_burg.py

```diff
@@ -0,0 +1,110 @@
+import numpy as np
+
+from statsmodels.tsa.innovations.arma_innovations import arma_innovations
+from statsmodels.tsa2.datasets.brockwell_davis_2002 import dowj, lake
+from statsmodels.tsa2.sarimax.estimators.burg import burg
+
+import pytest
+from numpy.testing import assert_allclose, assert_equal, assert_raises
+
+
+@pytest.mark.low_precision
+def test_brockwell_davis_example_513():
+    # Test against Example 5.1.3 in Brockwell and Davis (2016)
+    # (low-precision test, since we are testing against values printed in the
+    # textbook)
+
+    # Difference and demean the series
+    endog = dowj.diff().iloc[1:]
+
+    # Burg
+    res, _ = burg(endog, ar_order=1, demean=True)
+    assert_allclose(res.ar_params, [0.4371], atol=1e-4)
+    assert_allclose(res.sigma2, 0.1423, atol=1e-4)
+
+
+@pytest.mark.low_precision
```

Comment:
I like this idea.  How about taking it one step further and instead of putting the explanation in a comment, do:

```
@pytest.mark.low_precision("Test against Example 5.1.4 in Brockwell and Davis (2016) [...]")
```

Reply:
This is a good idea, thanks!